packages/SwingSet/src/liveslots/collectionManager.js

```diff
@@ -15,7 +15,7 @@ import {
   makeCopyMap,
 } from '@agoric/store';
 import { Far, passStyleOf } from '@endo/marshal';
-import { decodeToJustin } from '@endo/marshal/src/marshal-justin.js';
+// import { decodeToJustin } from '@endo/marshal/src/marshal-justin.js';
```

Comment:
You can just delete this line. There's no reason to go out of our way to preserve a Justin rendering here specifically.